projects/validation-messages/src/lib/resources/enums/validator-with-value.enum.ts

```diff
@@ -0,0 +1,5 @@
+export enum ValidatorWithValue {
```

Comment:
naming should include information that this is a default Angular Validator mapping

eg, AngularValidatorValue ?